assets/components/directDebit/directDebitPopUpForm/directDebitPopUpForm.jsx

```diff
@@ -0,0 +1,82 @@
+// @flow
+
+// ----- Imports ----- //
+
+import React from 'react';
+import { connect } from 'react-redux';
+import type { Currency } from 'helpers/internationalisation/currency';
+import { closeDirectDebitPopUp } from 'components/directDebit/directDebitActions';
+import DirectDebitForm from 'components/directDebit/directDebitForm/directDebitForm';
+
+
+// ---- Types ----- //
+
+/* eslint-disable react/no-unused-prop-types */
+type PropTypes = {
+  amount: number,
+  callback: Function,
```

Comment:
Could this have a more descriptive name?

Reply:
I am going to look deeper into the props (including props name) in the second phase of this feature development.